pkg/option/config.go

```diff
@@ -287,6 +287,9 @@ const (
 	// level annotation for configuring bpf loadbalancing algorithm.
 	LoadBalancerAlgorithmAnnotation = "bpf-lb-algorithm-annotation"
 
+	// LoadBalancerOnly is legacy knob for --datapath-mode=lb-only.
+	LoadBalancerOnly = "bpf-lb-only"
```

Comment:
is `bpf-lb-only` supported in combination with all datapath modes that are left (veth, netkit, netkitl2) ?

If not, do we need some validation?

Reply:
Yes this is completely independent of it, which is also why this should be moved out of `--datapath-mode`.